mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -93,6 +95,13 @@ class CrossValidator extends Estimator[CrossValidatorModel] with CrossValidatorP
   /** @group setParam */
   def setNumFolds(value: Int): this.type = set(numFolds, value)
 
+  override def validate(paramMap: ParamMap): Unit = {
+    val map = extractParamMap(paramMap)
+    getEstimatorParamMaps.foreach { eMap =>
+      getEstimator.validate(map ++ eMap)
```

Comment:
Should be `getEstimator.validate(eMap ++ paramMap)`. Estimator doesn't need CV params.